clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/Metadata.scala

```diff
@@ -106,13 +106,12 @@ object Metadata {
     * extension. Rather than make dangerous guesses, we require that the code calling
     * the move operation provide the file extension to retain during the move operation.
     */
-  def findNewPathForMove(
+  def buildFilePath(
```

Comment:
Renamed because "find" made it sound like it was searching for an existing path instead of creating a new one, and because it's not just used for file moves.